pkgs/top-level/perl-packages.nix

```diff
@@ -1973,6 +1973,23 @@ let self = _self // overrides; _self = with self; {
     };
   };
 
+  CryptOpenSSLAES = buildPerlPackage rec {
+    name = "Crypt-OpenSSL-AES-0.02";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/T/TT/TTAR/${name}.tar.gz";
+      sha256 = "b66fab514edf97fc32f58da257582704a210c2b35e297d5c31b7fa2ffd08e908";
+    };
+    NIX_CFLAGS_COMPILE = "-I${pkgs.openssl}/include";
```

Comment:
Unfortunately it doesn't seem to work with just `buildInputs` or even `makeMakerFlags`. This package has a very simple `Makefile.PL` that doesn't offer much in the way of providing an alternate library path. It's possible to provide the include path using `makeMakerFlags` so an alternative might be to have
```
    makeMakerFlags = "INC=-I${pkgs.openssl}/include";                  
    NIX_CFLAGS_LINK = "-L${pkgs.openssl}/lib -lcrypto"; 
```
I don't know if that offer much of an improvement, though.